CONTRIBUTING.rst

```diff
@@ -0,0 +1,35 @@
+Contributing to Renku
+=====================
+
+Want to contribute to Renku? Thanks! There are many ways to help, and we very much
+appreciate your efforts.
+
+The sections below provide guidelines for various types of contributions.
+
+Bug Reports
+===========
+
+Bugs should be reported on GitHub: https://github.com/SwissDataScienceCenter/renku/issues
+
+When reporting a bug, please include the following information:
+
+- Your operating system name and version.
+- Any details about your local setup that might be helpful in troubleshooting.
+- Detailed steps to reproduce the bug.
+
+Pull Requests
+=============
+
+If you do not already have a development environment set up, you will probably find the
+`developer documentation`_ helpful.
+
+* Before submitting a pull request, please make sure you agree to the license and have submitted a signed `contributor license agreement`_.
+* PRs should include a short, descriptive title. The titles will be used to compile changelogs for releases, so think about the title in that context.
```

Comment:
> The titles will be used to compile changelogs for releases, so think about the title in that context.

Do we really want to use PR titles?

Reply:
We agreed IRL that at first we would do the changelogs by hand anyway from a combination of PR titles and commit messages. Still, pretending like the PR title will be used by a robot in a changelog might encourage better PR titles :)

Reply:
We should encourage ourselves to write nice commit message in case the commits are indexed in Renku itself one day ;)